src/main/resources/schema.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE IF NOT EXISTS crew (
+id BIGINT AUTO_INCREMENT ,
+name VARCHAR(50) NOT NULL ,
+
+CONSTRAINT crew_PK PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS airplane (
+id            BIGINT AUTO_INCREMENT ,
+code_name     VARCHAR(50) NOT NULL ,
+model         VARCHAR(50) NOT NULL ,
+manufacture_date      DATE NOT NULL ,
+capacity    INT NOT NULL ,
+flight_range DECIMAL NOT NULL ,
```

Comment:
just INT would be enough here

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/3d7c7d3d96fe1b8bd3698ee76a32a9f3d5b5354e